### PR TITLE
doc(exercise 4): transfer UST from Terra to EVM

### PR DESCRIPTION
**Pull request by haiyizxx**

### CODE REVIEW

**Review by Windforcer — APPROVED** [review]

**Review by milapsheth — APPROVED** [review]

File: documentation/docs/Exercises/Exercise 4.md [hunk 1]
```diff
@@ -59,95 +57,158 @@ source .profile
...
+
+1. On a new terminal window, enter Axelar node:
+```bash
+docker exec -it axelar-core sh
 ```
-Check that tokens have arrived
 
-[cosmoshub address] is the address you created in step 5, associated with the [cosmos-key-name]
+2. Create a deposit address on Axelar Network (to which you'll deposit coins later)
+[receipent addres] is the address you want to receive the tokens to
```
Comment (ggutoski) [inline documentation/docs/Exercises/Exercise 4.md:96]:
```suggestion
[receipent addres] is an address you control on the recipient EVM chain.  This is where your UST will ultimately be sent.
```
minor clarification

File: documentation/docs/Exercises/Exercise 4.md [hunk 2]
```diff
@@ -59,95 +57,158 @@ source .profile
...
-1. Send an IBC transfer from Cosmoshub testnet to Axelar Network
+Look for `successfully linked [Axelar Network deposit address] and [receipent address]`
 
-You can find `Cosmoshub channel id` under [Testnet Release](/testnet-releases)
+3. Send an IBC transfer from Terra testnet to Axelar Network
+Switch back to terminal with terrad installed
 
-[axelar address] is the address you generated in Exercise 3
+```bash
+terrad tx ibc-transfer transfer transfer [Terra channel id] [Axelar Network deposit address] --packet-timeout-timestamp 0 [amount]uusd --gas-prices 0.15uusd --from [terra-key-name] -y -b block
```
Comment (ggutoski) [inline documentation/docs/Exercises/Exercise 4.md:110]: duplicate `transfer`??

**Review by ggutoski — APPROVED** [review]

File: documentation/docs/Exercises/Exercise 4.md (hunk 1 above)
Comment (Windforcer) [reply documentation/docs/Exercises/Exercise 4.md:96]: Also a typo, "recipient address"

File: documentation/docs/Exercises/Exercise 4.md (hunk 2 above)
Comment (haiyizxx) [reply documentation/docs/Exercises/Exercise 4.md:110]: na, the first transfer is command name, the second transfer is port id

**Pull request merged**